backend/src/test/groovy/com/capstone/moneytree/utils/MoneyTreeTestUtils.groovy

```diff
@@ -60,6 +48,47 @@ class MoneyTreeTestUtils {
               .build()
    }
 
+   /**
+    * Utility method to create transactions
+    */
+   static List<Transaction> buildTransactions() {
+      return [
+         Transaction.builder().moneyTreeOrderType(MoneyTreeOrderType.MARKET_BUY).build(),
+         Transaction.builder().moneyTreeOrderType(MoneyTreeOrderType.MARKET_BUY).build(),
+         Transaction.builder().moneyTreeOrderType(MoneyTreeOrderType.MARKET_BUY).build(),
+         Transaction.builder().moneyTreeOrderType(MoneyTreeOrderType.MARKET_SELL).build(),
+         Transaction.builder().moneyTreeOrderType(MoneyTreeOrderType.LIMIT_BUY).build()
+      ]
+
+   }
+
+   /**
+    * Utility method to build an order
+    */
+   static buildMarketOrder() {
```

Comment:
I think we should pass in parameters here for symbol, quantity, etc to have generic tests.